com.unity.multiplayer.mlapi/CHANGELOG.md

```diff
@@ -19,7 +19,7 @@ This is the initial experimental Unity MLAPI Package, v0.1.0.
 
 - Added message batching to handle consecutive RPC requests sent to the same client. `RpcBatcher` sends batches based on requests from the `RpcQueueProcessing`, by batch size threshold or immediately.
 - [GitHub 494](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/494): Added a constraint to allow one `NetworkObject` per `GameObject`, set through the `DisallowMultipleComponent` attribute.
-- Integrated MLAPI with the Unity Profiler for versions 2020.2 and later:
+- Integrated MLAPI with the Unity Profiler for versions 2020.3 and later:
```

Comment:
I believe it should've been 2020.3 LTS instead of 2020.2 already — what you guys think? @lkrell @JesseOlmer